starter-code/main.js

```diff
@@ -0,0 +1,147 @@
+4.- List by name all companies founded in february of 2004.
+5.- List by name all companies founded in the summer of 2004 (april to june) sorted by date.
+6.- What companies have offices in "Barcelona".
+7.- List the 10 companies with more employees sorted ascending (show name and employees).
+8.- Find the company with the name "Facebook"
+9.- How many employees has Facebook?
+10.- List the name of all the products of Facebook
+11.- List the people that are working at Facebook right now (check relationships field)
+12.- How many people are not working anymore at Facebook
+13.- List all the companies where "david-ebersman" has worked.
+14.- List by name the competitors of Facebook
+15.- Names of the companies that has "social-networking" in tag-list (be aware that the value of field is a string check regex operators)
+16.- How many companies that has "social-network" in tag-list and founded between 2002 and 2016 inclusive
+17.- Names and locations of companies that have offices in London
+18.- How many companies that has "social-network" in tag-list and founded between 2002 and 2016 inclusive and has offices in New York
+`);
+}
+
+
+mongoClient.connect(url, (error, db) => {
+  if (error) {
+    console.log('Error trying to connect to the Database');
+    console.log(error);
+  } else {
+    console.log('Connection established correctly!! 😬');
+
+    function mainMenu(){
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch(option){
+          case "1":
+          db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+            }
+          }) 
+            break;
+        case "2":
+          db.collection('companies').find({}, {name: 1, _id: 0}).count((error, result) => {
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+            }});
+            break;
+        case "3":
+            db.collection('companies').find({"founded_year": 2004}, {name: 1, _id: 0}).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }});
+              break;
+        case "4":
+            db.collection('companies').find(
+                {$and: [{"founded_year": 2004},
```

Comment:
You don't need the $and here, when you put more than one query (for different criteria) separated by commas, a logic and is applied.